src/methods.ts

```diff
@@ -60,7 +60,7 @@ export interface Dialog {
     subtype?: 'email' | 'number' | 'tel' | 'url';
     // type `select`:
     data_source?: 'users' | 'channels' | 'conversations' | 'external';
-    selected_options?: string;
+    selected_options?: OptionField[];
```

Comment:
`OptionField[]` isn't actually right, since it defines the `text` property which cannot be used here. What we actually want is `SelectOption[]`

Reply:
👍 fixed!